tests/compas_timber/test_joint.py

```diff
@@ -69,59 +68,25 @@ def x_topo_beams():
 def test_create(mocker):
     mocker.patch("compas_timber.connections.Joint.add_features")
     # try create with beams
-    A = TimberAssembly()
-    B1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
-    B2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
-    A.add_beam(B1)
-    A.add_beam(B2)
-    J = TButtJoint.create(A, B1, B2)
-
-    assert len(list(A.graph.nodes())) == 3
-    assert len(list(A.graph.edges())) == 2
-    assert A.joints[0] == J
-
-
-def test_joint_beam_keys(mocker):
-    mocker.patch("compas_timber.connections.Joint.add_features")
-    # try create with beams
-    A = TimberAssembly()
-    B1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
-    B2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
-    A.add_beam(B1)
-    A.add_beam(B2)
-    J = TButtJoint.create(A, B1, B2)
-
-    assert len(list(A.graph.nodes())) == 3
-    assert len(list(A.graph.edges())) == 2
-    assert A.joints[0] == J
-
-
-def test_joint_override_protection(mocker):
-    mocker.patch("compas_timber.connections.Joint.add_features")
-    A = TimberAssembly()
-    B1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
-    B2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
-    B3 = Beam(Frame.worldZX(), length=1.0, width=0.1, height=0.1)
-    A.add_beam(B1)
-    A.add_beam(B2)
-    A.add_beam(B3)
-    J = TButtJoint.create(A, B1, B2)
-
-    assert A.are_parts_joined([B1, B2])
-    assert A.are_parts_joined([B1, B3]) is False
+    model = TimberModel()
+    b1 = Beam(Frame.worldXY(), length=1.0, width=0.1, height=0.1)
+    b2 = Beam(Frame.worldYZ(), length=1.0, width=0.1, height=0.1)
+    model.add_beam(b1)
+    model.add_beam(b2)
+    _ = TButtJoint.create(model, b1, b2)
 
-    A.remove_joint(J)
-    assert A.are_parts_joined([B1, B2]) is False
+    assert len(model.beams) == 2
+    assert len(model.joints) == 1
 
 
 def test_deepcopy(mocker, t_topo_beams):
     mocker.patch("compas_timber.connections.Joint.add_features")
-    assembly = TimberAssembly()
+    assembly = TimberModel()
```

Comment:
should we change variable names to match type name? Not important to me

Reply:
seems like a pain in the ass

Reply:
yes we should :)